resources/views/layout/metadata.blade.php

```diff
@@ -28,6 +28,24 @@
 <meta name="keywords" content="osu, peppy, ouendan, elite, beat, agents, ds, windows, game, taiko, tatsujin, simulator, sim, xna, ddr, beatmania, osu!, osume">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
+@if (isset($opengraph))
+    @if (isset($opengraph['section']))
+        <meta property="og:site_name" content="{{ 'osu! » '.$opengraph['section'] }}">
+    @else
+        <meta property="og:site_name" content="{{ 'osu!` }}">
```

Comment:
the content doesn't need to be inside `{{ }}`. And it's not even syntactically correct. Or make a `@php` block to set the correct `$siteName` and reduce duplicating the same property.